packages/p2p/src/components/my-profile/my-profile-form/my-profile-form.jsx

```diff
@@ -88,7 +88,9 @@ const MyProfileForm = () => {
         </Formik>
     );
 
-    return isDesktop() ? (
+    return my_profile_store.is_loading ? (
+        <Loading is_fullscreen={false} />
+    ) : isDesktop() ? (
```

Comment:
Eslint is going to throw a warning for the nested ternary operator. Instead you can just return it and wrap it in <DesktopWrapper> and <MobileWrapper> respectively.